hw02/PlayerController.cs

```diff
@@ -14,18 +25,51 @@ public class PlayerController : MonoBehaviour
                 1. Player or Character's Name
                 2. Movement Speed as a float
                 3. Gravity Value as a float
-                4. Jump Speed as a float
-    
+                4. Jump Speed as a float 
+    */
+
+    // Answer 2.1
+    // These are the variables that we will be using to control our player.
+    private string playerName = "Player";
+    private float movementSpeed = 10.0f;
+    private float gravityValue = 9.8f;
+    private float jumpSpeed = 10.0f;
+    // If we want change the name of player through the console, we can use the following code:
+    playername=Console.ReadLine();
```

Comment:
Close! In the Unity console, we can just use `Debug.Log`. Though `Console.ReadLine()` is good for the console in VSCode.